src/Screens/Explore/components/AutocompleteModal.tsx

```diff
@@ -51,7 +51,7 @@ const AutocompleteModal: FunctionComponent<ModalWFinalFormProps> = ({
   const [suggestions, setSuggestions] = React.useState<TypeaheadResponse[]>([]);
   const [loading, setLoading] = React.useState(false);
 
-  const makeRequest = debounce(async (val: string) => {
+  const makeRequest = React.useMemo(() => debounce(async (val: string) => {
```

Comment:
if you don't use this then the debounce doesn't work because the function gets recreated on each render